app/views/foreman_leapp/job_templates/preupgrade.erb

```diff
@@ -7,6 +7,13 @@ provider_type: SSH
 feature: leapp_preupgrade
 %>
 
+if ! yum list installed -q leapp
+then
+  echo "Leapp is not installed."
+  exit 1
+fi
```

Comment:
How about using `command -v leapp > /dev/null` instead? This would check that `leapp` is actually in user's `PATH` and not just that the package is installed